endpoints/README.MD

```diff
@@ -27,6 +27,107 @@ Note that not all notice types are supported in Hilma. Trying to create a non su
 
 Notice types E1 to E5 are not yet supported.
+  - Will be supported from 1.1.2024
+
+## Statistical questions
+
+- Statistical questions are asked only in procurement and contract-award notice types
+- Statistical questions are:
+  - Main statistical questions
+    - Preparation of the procurement(social and contract-award  procurements)
+  - Clean vehicles
+  - Accessability
+- Notices are visible in Hilma just like notices that are created in hilma web-application
+- Statistical questions that can be sent to TED, are visible in TED and HILMA
+  - If questions can not be sent to TED, then they are visible only in HILMA
+
+### Statistical questions in detail( yes/no)
+  - All questions are mandatory to answer
+
+#### Innovation
+  - In preparation, need and opportunities for innovation was considered.
+  - The solution sought after, or part of it, is new to us as a buyer. 
+  - The solution sought after, or part of it, is new to the market or industry. 
+
+#### Environmental sustainability
+  - Procurement takes into account energy efficiency
+    - Energy-using equipment and solutions, and construction
+  - This procurement promotes low carbon emissions. 
+    - Emission reduction; reducing the production and use of fossil energy or increasing the production and use of renewable energy.
+  - This procurement promotes circular economy.
+  - This procurement promotes biodiversity
+  - This procurement promotes a sustainable food system.
+    - Promotion of good farming practices, promotion of food safety, promotion of animal welfare.
+  - Are Motiva's, ecolabels' or EU GPP criteria used in the procurement? 
+
+#### Social sustainability
+  - This procurement promotes fair working conditions. 
+  - This procurement includes a requirement to provide employment for people with partial work capacity
+  - Code of Conduct is used in this procurement. 
+
+#### Consideration of SMEs
+  - Procurement allows for participation of small and medium-sized enterprises.
+
+## Example json payload for hilma statistics
+
+- Note: key for statistics payload is LOT id
+  - LOT ID is defined in Eform's json:
+  ```json
+  "ProcurementProjectLot": [
+    {
+      ...
+      "ID": { "schemeName": "Lot", "Value": "LOT-0000" }
+      ...
+    }
+  }
+  ```
+
+```json
+{
+  "LOT-0000": {
+    "EnergyEfficiencyConsidered": false,
+    "LowCarbon": true,
+    "CircularEconomy": true,
+    "Biodiversity": false,
+    "SustainableFoodProduction": false,
+    "ListedGreenCriteriaUsed": true,
+    "JustWorkingConditions": true,
+    "EmploymentCondition": false,
+    "HowManyOpportunitiesIsEstimated": false,
+    "CodeOfConduct": true,
+    "InnovationConsidered": true,
+    "SolutionNewToBuyer": true,
+    "SolutionNewToMarketOrIndustry": false,
+    "SMEParticipationConsidered": false,
+    "EndUserInvolved": true,
+  }
+}
+
+```
+
+## Hilma to TED mapping values. (hilma key -> TED value)
+- TED field: **bt-776**
+- codelist: **innovative-acquisition**
```

Comment:
Added documentation for specific hilma -> TED mapping values in statistics questions